src/vm/excep.cpp

```diff
@@ -5403,7 +5403,7 @@ LONG InternalUnhandledExceptionFilter_Worker(
 #endif // DEBUGGING_SUPPORTED
 
 
-#ifdef FEATURE_EVENT_TRACE    
+#if defined(FEATURE_EVENT_TRACE) && !defined(FEATURE_PAL)  //Enable once Eventrepoter is ported Cross Plat
```

Comment:
Please add UNIXTODO to the comment, we use it to quickly locate changes that are planned to be made for Unix